From 22e852d1eff72179c00d88183eac0187d1fd9048 Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.de>
Date: Wed, 23 Oct 2013 11:25:33 +0200
Subject: [PATCH 2/3] xhci-hub.c: preserved kABI
Patch-Mainline: Never
References: bnc#840116
Git-Commit: 22e852d1eff72179c00d88183eac0187d1fd9048

needed because an earlier patch wasn't taken for kABI reasons

Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
 drivers/usb/host/xhci-hub.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/usb/host/xhci-hub.c b/drivers/usb/host/xhci-hub.c
index f43f4a0..75b8675 100644
--- a/drivers/usb/host/xhci-hub.c
+++ b/drivers/usb/host/xhci-hub.c
@@ -585,7 +585,6 @@ static u32 xhci_get_port_status(struct usb_hcd *hcd,
 			xhci_dbg(xhci, "Resume USB2 port %d\n",
 					wIndex + 1);
 			bus_state->resume_done[wIndex] = 0;
-			clear_bit(wIndex, &bus_state->resuming_ports);
 			xhci_set_link_state(xhci, port_array, wIndex,
 					XDEV_U0);
 			xhci_dbg(xhci, "set port %d resume\n",
-- 
1.8.3.1

